modules/accounts/src/lib.rs

```diff
@@ -901,3 +890,25 @@ where
 		Ok(())
 	}
 }
+
+impl<T: Trait> MergeAccount<T::AccountId> for Module<T> {
+	fn merge_account(source: &T::AccountId, _dest: &T::AccountId) -> DispatchResult {
+		Self::do_merge_account_check(source)
+	}
+}
+
+pub struct CallKillAccount<T>(PhantomData<T>);
+impl<T: Trait> OrmlHappened<T::AccountId> for CallKillAccount<T> {
+	fn happened(who: &T::AccountId) {
+		if system::Account::<T>::contains_key(who) {
```

Comment:
Should just change to  `frame_system::CallKillAccount::happened(who)` so we don't need to duplicate code